### PR TITLE
netspeed: Add NetspeedPreferences class

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

**Review by lukefromdc — APPROVED** [review]
This builds with only deprecation errors, and runs fine. No problems bringing up  or using the netspeed applet's preference dialog

File: netspeed/src/netspeed.c [hunk 1]
```diff
@@ -1059,117 +1068,14 @@ about_cb (GtkAction *action,
...
 
-/* this basically just retrieves the new devicestring
- * and then calls applet_device_change () and change_icons ()
- */
 static void
-device_change_cb (GtkComboBox    *combo,
-                  NetspeedApplet *netspeed)
+netspeed_applet_destory_preferences (GtkWidget *widget,
+                                     gpointer   user_data)
 {
```
Comment (raveit65) [inline netspeed/src/netspeed.c:1074]: New build warning.
```
netspeed.c:1072:49: warning: unused parameter 'widget' [-Wunused-parameter]

netspeed_applet_destory_preferences (GtkWidget *widget,
```
https://travis-ci.org/github/mate-desktop/mate-applets/jobs/751446165#L6908

**Comment by raveit65** [comment]
Beside from this warning the applet runs fine and dialogs are looking like before.

File: netspeed/src/netspeed.c (hunk 1 above)
Comment (rbuj) [reply netspeed/src/netspeed.c:1074]: done

**Review by raveit65 — APPROVED** [review]
LGTM,
builds and runs fine, and new build warning is fixed.
Thx.

**Comment by lukefromdc** [comment]
If that warning is cut and pasted we also have a (cosmetic) typo in the name of a function(destory)

**Comment by rbuj** [comment]
netspeed_applet_destory_preferences 😄

**Comment by rbuj** [comment]
@lukefromdc thank you very much for the find, PR #588

**Pull request merged**